scripts/Calibration/Examples/TubeCalibDemoMaps_All.py

```diff
@@ -500,8 +502,8 @@ def completeCalibration(filename):
     mantid.ApplyCalibration(Workspace=CalibInstWS, PositionTable=calibrationTable)
 
     # == Save workspace ==
-    # SaveNexusProcessed( CalibInstWS, path+'TubeCalibDemoMapsResult.nxs',"Result of Running TCDemoMaps.py")
-    # print "saved calibrated workspace (CalibInstWS) into Nexus file TubeCalibDemoMapsResult.nxs"
+    # mantid.SaveNexusProcessed(CalibInstWS, path+'TubeCalibDemoMapsResult.nxs', "Result of Running TCDemoMaps.py")
```

Comment:
If it's going to be used for teaching, it probably makes sense to leave them in. The `print` statements probably want to use the Python 3 syntax, though